web/source/osk/visualKeyboard.ts

```diff
@@ -207,7 +216,20 @@ namespace com.keyman.osk {
         this.inputEngine.registerEventHandlers();
       }
 
-      Lkbd.className = device.formFactor + ' kmw-osk-inner-frame';
+      Lkbd.classList.add(device.formFactor);
+      Lkbd.classList.add('kmw-osk-inner-frame');
```

Comment:
```suggestion
      Lkbd.classList.add(device.formFactor, 'kmw-osk-inner-frame');
```

See [spec](https://developer.mozilla.org/en-US/docs/Web/API/DOMTokenList/add).